### PR TITLE
Breaking snitches now displays who owns them and what group they were on

### PR DESCRIPTION
**Pull request by TealNerd**

### CODE REVIEW

File: src/com/untamedears/JukeAlert/listener/JukeAlertListener.java [hunk 1]
```diff
@@ -301,6 +301,14 @@ public void breakSnitchBlock(BlockBreakEvent event) {
             return;
         }
         Block block = event.getBlock();
+        if(block.getType().equals(Material.JUKEBOX) || block.getType().equals(Material.NOTE_BLOCK) && plugin.getConfigManager().isDisplayOwnerOnBreak()) {
```
Comment (rourke750) [inline src/com/untamedears/JukeAlert/listener/JukeAlertListener.java]: Dont think this logic is right.  Shouldn't it be 

> if((block.getType().equals(Material.JUKEBOX) || block.getType().equals(Material.NOTE_BLOCK)) && plugin.getConfigManager().isDisplayOwnerOnBreak())

File: src/com/untamedears/JukeAlert/manager/ConfigManager.java [hunk 2]
```diff
@@ -112,7 +113,8 @@ private void load()
         alertRateLimit = loadInt("settings.alertratelimit", 70);
         enableInvisibility = loadBoolean("settings.enableinvisiblity", false);
         toggleRestartCheckGroup = loadBoolean("settings.togglerestartgroupcheck", false);
-        
+        displayOwnerOnBreak = loadBoolean(“settings.displayOwnerOnSnitchBreak”, true);
```
Comment (rourke750) [inline src/com/untamedears/JukeAlert/manager/ConfigManager.java]: Um, not sure what you did there, those aren't parenthesis.

**Comment by TealNerd** [comment]
The logic should still work but i can add the extra parenthesis just to be sure. as for the config thing, those are definitely parenthesis...

**Comment by rourke750** [comment]
For the if statement lets say some breaks a Jukebox wouldnt and config was set to false: true || false && false == true || false making the statement true even though it should be false or am I messing up?

**Comment by rourke750** [comment]
ok to test

**Comment by rourke750** [comment]
ok to test

**Comment by rourke750** [comment]
ok to test

**Comment by TealNerd** [comment]
Also I think you meant to say they weren't quotes not they weren't parenthesis 
@rourke750

**Comment by rourke750** [comment]
Lol yep.

**Comment by rourke750** [comment]
ok to test

**Comment by TealNerd** [comment]
@rourke750 can we merge this?

**Comment by Maxopoly** [comment]
Did you test this?

**Comment by ttk2** [comment]
bump.

**Comment by TealNerd** [comment]
yes

**Pull request merged**